tests/converter/test_consistency_check.py

```diff
@@ -114,7 +114,7 @@ def test_conv_consistency(self) -> None:
             [1, 3, 3, 3],
             Float32(),
             input_ops,
-            pads=[1, 1, 2, 2],
+            pads=[1, 2, 1, 2],
```

Comment:
@primenumber 
Sorry, I can not understand this change clear.
pad format is `[top, bottom, left, right]` and this line means using different padding value for top and buttom. right?